MakieCore/src/types.jl

```diff
@@ -1,17 +1,9 @@
 
-"""
-    abstract type Transformable
-This is a bit of a weird name, but all scenes and plots are transformable,
-so that's what they all have in common. This might be better expressed as traits.
-"""
-abstract type Transformable end
-
-abstract type AbstractPlot{Typ} <: Transformable end
-abstract type AbstractScene <: Transformable end
-abstract type ScenePlot{Typ} <: AbstractPlot{Typ} end
-abstract type AbstractScreen <: AbstractDisplay end
-
-const SceneLike = Union{AbstractScene, ScenePlot}
+const TorVector{T} = Union{T, AbstractVector{T}}
```

Comment:
Since this is used in `struct` fields, it sure would be nice to make this `Vector{T}` instead of `AbstractVector` if possible.

Reply:
Yaaah... 😢 I planned to Benchmark this when the PR is in a better state - but would be sad to give up on AbstractArray support, but maybe we can make it work one way or the other...

Reply:
You can of course have both `TorAbstractVector` and `TorVector`.